iOS/WAStickersThirdParty/ImageData.swift

```diff
@@ -161,6 +165,10 @@ class ImageData {
                 throw StickerPackError.incorrectImageSize(imageData.image!.size)
             }
         } else {
+            guard imageData.bytesSize > 0 else {
+                throw StickerPackError.imageTooSmall
+            }
+
```

Comment:
You can just move this guard before the if condition since it's the same for both cases